README.md

```diff
@@ -54,10 +54,13 @@ Next, either download a release or clone this repo and do the following:
 
 ✅ `./run-platform.sh`<br>
 ✅ Now visit [http://frontend.unstract.localhost](http://frontend.unstract.localhost) in your browser <br>
-✅ Use user name and password `unstract` to login
+✅ Use username and password `unstract` to login
+
 
 That's all there is to it!
 
+See [this guide](backend/README.md#authentication) for steps to change the default username and password.
+
 See [user guide](https://docs.unstract.com/unstract/unstract_platform/user_guides/run_platform) for more details on managing the platform.  
 Another really quick way to experience Unstract is by signing up for our [hosted version](https://us-central.unstract.com/).
```

Comment:
```suggestion
Another really quick way to experience Unstract is by signing up for our [hosted version](https://us-central.unstract.com/). It comes with a 14 day free trial!
```
NIT: Added this suggestion to nudge the OSS user slightly towards trying our hosted version